src/FilmCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+type Film = {
```

Comment:
Паттерн нейминга ИмяКомпонентаProps
```suggestion
type FilmCardProps = {
```